loader.js

```diff
@@ -54,19 +57,25 @@ module.exports = function() {
   const cwd = dirname(this.resourcePath);
   const pkgDir = join(cwd, './pkg');
 
+  log(`:information_source:  Compiling your Rust code...`, 'green');
```

Comment:
Instead of rust code or library I think that `crate` is the correct term (Cargo.toml is its manifest).

Reply:
changed!